### DISCUSSION TITLE
Is it possible to have multiple worth.yml files ?

### DISCUSSION THREAD

**Discussion by Spygain** [Zrips/CMI#5745] [Q&A]
Hello,

I come to you to know if it is possible to have multiple worth.yml files, to make different shop in my server

I am currently under 1.16.5 in bungeecord but I need it just for one server

Thank you in advance and I hope to have been clear enough even if the question is quite simple

Thanks in advance for the answer

**Comment by mrfloris** [comment]
hi there, sorry no (but maybe, see below), as far as I am aware there's one worth.yml file that's being used. 

As far as I am aware worth.yml isn't shared over bungee by CMI, if you have 3 servers and give them each a different worth.yml file then they all have a different values. They don't sync-over-bungee. If they do, i'd love to know and I will update my bungee documentation.
